Patch from upstream release 41.3

From 281bade14c1925cf9e7329fa8e9cf2d82512c66f Mon Sep 17 00:00:00 2001
From: Emmanuele Bassi <ebassi@gnome.org>
Date: Mon, 24 Jan 2022 23:39:24 +0000
Subject: [PATCH] build: Fix i18n.merge_file() use with Meson 0.61

The function never took positional arguments.

Fixes: #59
---
 data/meson.build | 2 --
 1 file changed, 2 deletions(-)

diff --git a/data/meson.build b/data/meson.build
index c6aeffb4..0bc531a2 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -19,7 +19,6 @@ appdata_conf = configuration_data()
 appdata_conf.set('application_id', APPLICATION_ID)
 appdata = APPLICATION_ID + '.appdata.xml'
 appdata_file = I18N.merge_file(
-  appdata,
   input: configure_file(
     input: 'org.gnome.Devhelp.appdata.xml.in.in',
     output: APPLICATION_ID + '.appdata.xml.in',
@@ -45,7 +44,6 @@ desktop_conf = configuration_data()
 desktop_conf.set('application_id', APPLICATION_ID)
 desktop = APPLICATION_ID + '.desktop'
 desktop_file = I18N.merge_file(
-  desktop,
   type: 'desktop',
   input: configure_file(
     input: 'org.gnome.Devhelp.desktop.in.in',
-- 
GitLab

